displayio/display.py

```diff
@@ -53,8 +53,17 @@
 BACKLIGHT_PWM = 2
 
 # pylint: disable=unnecessary-pass, unused-argument
-
 # pylint: disable=too-many-instance-attributes
+
+
+def _rgb_tuple_to_rgb565(color_tuple):
+    return (
+        ((color_tuple[0] & 0x00F8) << 8)
+        | ((color_tuple[1] & 0x00FC) << 3)
+        | (color_tuple[2] & 0x00F8) >> 3
+    )
```

Comment:
There's already a function that does this in ColorConverter. Can you try making use of that? I'm not certain if you would need to make adjustments, but it should work with tuples and convert it in a similar manner.

Reply:
Neat. I missed that, I really should poke around the existing code some more. Thank you. Will change it to use the existing function.